app/workers/my_anime_list_sync_worker.rb

```diff
@@ -1,10 +1,20 @@
 class MyAnimeListSyncWorker
   include Sidekiq::Worker
 
-  def perform(library_entry_id, method)
+  # data is a hash
+  def perform(data)
```

Comment:
"Almost the same" can be a million times worse than "completely different".  Either make it the same, or make it different.  A muddy middle ground is confusing